sample/src/androidTest/java/com/schibsted/spain/barista/sample/RadioButtonsTest.java

```diff
@@ -55,4 +58,9 @@ public void checkRadioButtonsByPosition_secondItem() {
     clickRadioButtonPosition(R.id.radiogroup, 1);
     assertDisplayed("" + R.id.second_item);
   }
+
+  @Test
+  public void checkRadioButtonContainsText() {
+    assertContains("bye");
```

Comment:
>I don't think this test should be part of this class. 
The `assertContains` method belongs to the `AssertionsTest`. I don't see the need to specifically add it to RadioButtons test.

Same here

Reply:
Done